python/src/typechat/_internal/ts_conversion/python_type_to_ts_nodes.py

```diff
@@ -234,70 +246,93 @@ def convert_to_type_node(py_type: object) -> TypeNode:
         errors.append(f"'{py_type}' cannot be used as a type annotation.")
         return AnyTypeReferenceNode
 
-    def declare_property(name: str, py_annotation: type | TypeAliasType, optional: bool):
-        origin: object = py_annotation
-        comments: str = ""
-        while origin := get_origin(origin):
-            if origin is Annotated and hasattr(py_annotation, "__metadata__"):
-                first_doc = next((x for x in py_annotation.__metadata__ if isinstance(x, Doc)), None)
-                if first_doc:
-                    comments =  first_doc.documentation
+    def declare_property(name: str, py_annotation: type | TypeAliasType, is_typeddict_attribute: bool, optionality_default: bool):
+        """
+        Declare a property for a given type.
+        If 'optionality_default' is 
+        """
+        current_annotation: object = py_annotation
+        origin: object
+        optional: bool | None = None
+        comment: str | None = None
+        while origin := get_origin(current_annotation):
+            if origin is Annotated and comment is None:
+                current_annotation = cast(Annotatedish, current_annotation)
+                annotation_metadata = current_annotation.__metadata__[0]
```

Comment:
There could be multiple annotations on a type and the comment not guaranteed to be at index 0.  The previous implementation looked through all the metadata entries and found the first one that was either a Doc or a str. We probably want to keep that behavior

Reply:
It's super questionable to have more than one piece of metadata, but sure, I can go back to finding the first.